docs/enrollment_and_payment.md

```diff
@@ -0,0 +1,138 @@
+## Enrolling a card
+
+In order to perform a payment a card first needs to be enrolled and tokenized. First you must gather the necessary information to perform an enrollment. This data is then sent to the ePayment Platform for tokenization.
+The account number and NIN is used to identify the card that is to be tokenized. The EPP will then asynchronously send a callback to the Integrator's Callback Server with the result of the enrollment, as well as any lifecycle changes.
+
+The resulting Payment Token is then used as a reference to the account in subsequent payment requests.
+
+In the case of a end customer revoking/deleting their payment source a request should be sent to the ePayment Platform to delete the token.
+
+### Tokenization
+
+Once you have requested a token you should expect the asynchronous call to your Callback Server. Your Callback Server must comply with our
+[Integration Specification](/assets/swagger/swagger_integrator_token_requestor_bankaxept/).
+
+The tokenization request must include a `tokenRequestorReference`. A reference set by the Token Requestor to uniquely identify an enrolment request. 
```

Comment:
Comma or "It is" before starting the sentence here?

Reply:
Adjusted!